xdsl/irdl.py

```diff
@@ -325,6 +325,36 @@ def irdl_to_attr_constraint(
 
 class IRDLOperation(Operation):
 
+    @staticmethod
+    def get_def() -> OpDef:
```

Comment:
feels like we should move `irdl_definition` here instead

Reply:
Oh, that was confused indeed. Good catch, thanks!

Reply:
Done!